cli/src/commands/cloud/environments/tunnel/mod.rs

```diff
@@ -32,19 +33,15 @@ mod renderer;
 #[cling(run = "run_tunnel")]
 #[clap(visible_alias = "expose", visible_alias = "tun")]
 pub struct Tunnel {
-    /// The port of a local service to expose to the Environment
-    #[clap(short = 'l', long)]
-    local_port: Option<u16>,
+    /// If set, do not expose local ports to the Cloud Environment
+    #[clap(long = "no-local")]
```

Comment:
Is this a bit ambiguous? Maybe "inbound" is more intuitive?

```suggestion
    /// Disable forwarding requests from the Cloud Environment to egress from the local tunnel
    #[clap(long = "no-inbound")]
```